src/ClassFinder.php

```diff
@@ -153,12 +153,9 @@ public static function getClasses(string $namespace = '', callable $conditional
             $conditional,
             $includeVendor
         ) {
-            /*$dontSkip = true;
-            if (!$includeVendor) {
-                $dontSkip = !self::isClassInVendor($class);
-            }*/
-            $dontSkip = $includeVendor || !self::isClassInVendor($class);
-            return substr($class, 0, strlen($namespace)) === $namespace && $dontSkip && $conditional($class) ;
+            return self::strStartsWith($namespace, $class) && 
```

Comment:
Is this strStartsWith check added to make it stone obvious we're only wanting things in $namespace?

Reply:
Get project classes autoload in the namespace is then return ALL declared classes (so we can cache the loaded namespace)

We need to filter the result of get_declared_classes